index.html

```diff
@@ -18,6 +18,15 @@
       rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200..1000;1,200..1000&display=swap"
     />
+
+    <!-- Umami tracking code -->
+    <!-- TODO: use an env var -->
```

Comment:
Todo now?

Reply:
Done. I think it can be done better, by for example defining a custom vite plugin that could insert the tag if the variable is defined. Here is an example: https://github.com/KermanX/vite-plugin-ms-clarity/blob/main/plugin.ts